### PR TITLE
Add info msg when logging ns likely wrong

### PR DESCRIPTION
**Pull request by mmazur**
This would help out in responding to some of the es issues.

### CODE REVIEW

**Comment by mmazur** [comment]
Hmm, on the other hand maybe keep the error detection and suggestion msg, but drop the es component part and leave that for separate branches.

Wonder how many things actually differ between the branches…

File: scripts/allocate-empty-primary [hunk 1]
```diff
@@ -15,7 +15,7 @@ source env-prep
 
 node=${1:-}
 if [ -z "${node}" ] ; then
-  node=$(oc -n $LOGGING_NS get dc -l component=es -o jsonpath={.items[0].metadata.name})
+  node=$(oc -n $LOGGING_NS get dc -l component=$ES_COMPONENT -o jsonpath={.items[0].metadata.name})
```
Comment (jcantrill) [inline scripts/allocate-empty-primary]: This depends on which version you are using 3.x vs 4.x.  It's different in both versions but constant to a particular version.  I would revert this.

File: scripts/env-prep [hunk 2]
```diff
@@ -4,11 +4,24 @@ if [ -n "${SDEBUG:-}" ] ; then
...
+# Override default logging namespace
+# Try 'openshift-logging' or 'logging'
 if [ -f ".logging-ns" ] ; then
   LOGGING_NS=$(cat .logging-ns)
 fi
 
+# Override default elasticsearch component
+# Try 'elasticsearch' or 'es'
+if [ -f ".es-component" ] ; then
+  ES_COMPONENT=$(cat .es-component)
```
Comment (jcantrill) [inline scripts/env-prep]: I'm not sure it makes sense to do this given 3.x and 4.x deployments are different and there is some drift.

**Comment by mmazur** [comment]
Ok, dropped the ES stuff, just kept the info msg for wrong ns. Not sure if this makes sense on master, but by that logic, the ns overriding in general likely doesn't, so I guess this might as well get merged.

Will you add this to the 3.x branch or should I send a separate PR?

**Comment by jcantrill** [comment]
> Ok, dropped the ES stuff, just kept the info msg for wrong ns. Not sure if this makes sense on master, but by that logic, the ns overriding in general likely doesn't, so I guess this might as well get merged.
> 
In 3.x this was different because we allowed you to modify the NS where logging was deployed with a preference to 'openshift-logging'  Additionally, earlier deployments the default was 'logging'.

> Will you add this to the 3.x branch or should I send a separate PR?

Please open a separate PR if you believe it is required.

**Pull request merged**